websites/B/Bunpro/presence.ts

```diff
@@ -41,15 +41,15 @@ class Timer {
     this.elapsed = 0;
     this.running = false;
 
-    localStorage.removeItem("premid__timestamp");
+    sessionStorage.removeItem("PMD_timestamp");
   }
 
   start() {
     if (!this.running) {
-      this.elapsed = Math.round(Date.now() / 1000);
+      this.elapsed = Date.now();
```

Comment:
Huh? It was okay. I just meant that you don't need this timer function at all because you can just replace startTimestamp :///